docs/features/importing_n_files.md

```diff
@@ -20,6 +20,9 @@ let importedThings = imp "./import.n" // This goes though the file so it will pr
 print(importedThings.test) // Prints out hello
 //print(importedthigns.test2) will cause an error because it is not public
 ```
+The `pub` modifier is used to export values to other files. This is why it can be applied to nearly every type or variable declaration, as by default, all values are private, to prevent malicous or accidential messing of code in other files.
```

Comment:
```suggestion
The `pub` modifier is used to export values to other files. This is why it can be applied to nearly every type or variable declaration, as by default, all values are private, to prevent malicious or accidental messing of code in other files.
```